### PR TITLE
Error 2001 - App does not work anymore (EXPOSUREAPP-2009)

### PR DESCRIPTION
**Pull request by LukasLechnerDev**
Users that deactivate the certificate `T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2` receive an "Error 2001" and see this error dialog: 

Error2001-Before

This PR sets a custom message to the error dialog: 

Error2001-After

Initially, it was planned to add a link to the faq entry of this error (https://www.coronawarn.app/en/faq/#cause2001). However, since the website on which the FAQs are hosted uses the same (deactivated) certificate, the browser won't let the user access this site. 

I clarified this issue with Thomas Augsten and we adapted the error message.

How to test: Open Settings - "Biometrics and Security" - "Other Security Settings" - "View security certificates" and deactivate the following certificate: 

deactivate_certificate

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1408,6 +1408,7 @@
     <string name="errors_not_enough_device_storage">"You do not have sufficient free memory."</string>
     <!-- XTXT: error dialog - detailed text if there is error with Google API  -->
     <string name="errors_communication_with_api">"Error during communication with Google interface"</string>
+    <string name="errors_communication_with_api_test">"Test ABC"</string>
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/main/res/values/strings.xml:1411]: Relict of some testing? Please remove it.

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1418,6 +1419,9 @@
     <string name="errors_risk_detection_limit_reached_title">"Limit already reached"</string>
     <!-- XTXT: error dialog - Error description when the provideDiagnosisKeys quota limit was reached. -->
     <string name="errors_risk_detection_limit_reached_description">"No more exposure checks possible today, as you have reached the maximum number of checks per day defined by your operating system. Please check your risk status again tomorrow."</string>
+    <!-- XTXT: error dialog - Error description when the ssl certificate is deactivated on the device. -->
+    <string name="errors_ssl_certificate_deactivated">Bitte aktiviere das SYSTEM Sicherheitszertifikat T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2 auf ihrem Gerät. Mehr Informationen finden sie im FAQ auf https://coronawarn.app"</string>
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/main/res/values/strings.xml:1423]: Don't we have a deep link to a specific question and answer?

**Review by ralfgehrer — DISMISSED** [review]

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 1 above)
Comment (LukasLechnerDev) [reply Corona-Warn-App/src/main/res/values/strings.xml]: Removed

**Review by ralfgehrer — DISMISSED** [review]

**Review by BMItr — DISMISSED** [review]

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -1403,6 +1403,9 @@
     <string name="errors_risk_detection_limit_reached_title">"Limit bereits erreicht"</string>
     <!-- XTXT: error dialog - Error description when the provideDiagnosisKeys quota limit was reached. -->
     <string name="errors_risk_detection_limit_reached_description">"Heute sind keine weiteren Risiko-Überprüfungen möglich, weil das von Ihrem Betriebssystem festgelegte Limit von Risiko-Überprüfungen pro Tag bereits erreicht ist. Bitte überprüfen Sie Ihren Risikostatus morgen wieder."</string>
+    <!-- XTXT: error dialog - Error description when the ssl certificate is deactivated on the device. -->
+    <string name="errors_ssl_certificate_deactivated">Bitte aktiviere das SYSTEM Sicherheitszertifikat T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2 auf ihrem Gerät. Mehr Informationen finden sie im FAQ auf https://coronawarn.app</string>
```
Comment (PhilippNowak96) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:1407]: Mix-up between `du` and `Sie` concerning `Bitte aktiviere ... auf ihrem Gerät`

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1418,6 +1418,9 @@
...
```
Comment (PhilippNowak96) [inline Corona-Warn-App/src/main/res/values/strings.xml:1422]: siehe oben

File: Corona-Warn-App/src/main/res/values-de/strings.xml (hunk 3 above)
Comment (janetback) [reply Corona-Warn-App/src/main/res/values-de/strings.xml]: Hi @ralfgehrer , I suggest the following wording: 
Bitte aktivieren Sie das SYSTEM Sicherheitszertifikat T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2 auf Ihrem Gerät. Mehr Informationen finden Sie in den FAQs auf https://coronawarn.app unter „URSACHE 2001“.

File: Corona-Warn-App/src/main/res/values/strings.xml (hunk 4 above)
Comment (ralfgehrer) [reply Corona-Warn-App/src/main/res/values/strings.xml:1422]: This will be overwritten by the next translation delivery with the English version.

**Review by janetback — DISMISSED** [review]

**Review by ralfgehrer — APPROVED** [review]
Tested on Pixel (Android 10)
Screenshot 2021-01-25 at 12 53 59

**Review by BMItr — APPROVED** [review]

**Comment by MikeMcC399** [comment]
This PR addresses the issue from #968.

**Pull request merged**